src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
 // write code here
+const ulItem = document.querySelector('ul');
+const liItems = document.querySelectorAll('li');
+
+function sortList(list, listItems) {
+  const itemsSorted = [...listItems].sort((a, b) => (
+    +((b.dataset.salary)
+      .slice(1)
+      .split(',')
+      .join(''))
+    - +((a.dataset.salary)
+      .slice(1)
+      .split(',')
+      .join(''))
+  ));
+
+  list.innerHTML = '';
+
+  for (let i = 0; i < itemsSorted.length; i++) {
+    list.appendChild(itemsSorted[i]);
+  }
+}
+
+function getEmployees(listItems) {
+  const result = [];
```

Comment:
In this case you'll store eployees objects in this array, so better to specify that in name